app/controllers/posts_controller.rb

```diff
@@ -7,5 +7,6 @@ def index
   def show
```

Comment:
- [ ] ### I am leaving this comment here since there is no change on database, but your app experiencing an error when migrating databases, kindly fix that:
- ![image](https://user-images.githubusercontent.com/62705947/214438951-ea686644-86c5-4eab-aba5-c780e1a45cef.png)

Reply:
Hello @GzimAsani I don't know what exactly the issue is about because `rails db:migrate` runs perfectly for me. I just tested it.

Reply:
> Hello @GzimAsani I don't know what exactly the issue is about because `rails db:migrate` runs perfectly for me. I just tested it.

Hi @didierganthier, please check if you have any uncommitted code. If not, and if it works for you please ask for another code review and another reviewer can check it, maybe it was something wrong on my laptop. But I believe the error may be from the undefined t.reference that you have used on migrations instead of add_reference

Reply:
I don't but I'll run additional tests and I'll submit a video of the demo if necessary. Thanks @GzimAsani 👍🏽